README.md

```diff
@@ -2,7 +2,14 @@
 
 > Modern CSS linter.
 
-Supports the latest CSS syntax and features, including custom properties, range context for media features, calc() and nesting.
+## Features
+
+* _Over 85 rules_ - from stylistic rules, such as checking the spacing around the colon in every declaration, to rules that catch subtle coding mistakes, such as invalid hex colors or [overriding shorthand properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties#Tricky_edge_cases).
```

Comment:
I think this number is not a good idea and I would go for a approximation like "almost a hundred of rules" (to avoid having to update it.